source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -37,7 +37,28 @@ Covariance-based method
 ----------------
 SVD-based method
 ----------------
-[TBD]
+
+This method uses singular value decomposition of the dataset to find its
+principal components. The following steps are distinguished:
+
+#. Singular values and singular vector computation
+#. Resulting matrix formation
+
+To compute singular values and singular vectors, the following equation system
+shall be resolved:
+
+.. math::
+   \begin{cases}
+      X v = \lambda u, \\
```

Comment:
Do not discuss svd, just refer to the bibliography containing description of the svd methods. Implementation is free to choose any svd algorithm
The same generally applies to eigenvalue decomposition in covariance method

Reply:
fixed